src/Utilities/Requires.hpp

```diff
@@ -11,7 +11,7 @@
 namespace Requires_detail {
 template <bool B>
 struct requires_impl {
-  using template_error_types_failed_to_meet_requirements_on_template_parameters =
+  using template_error_types_failed_to_meet_requirements_on_template_parameters=
```

Comment:
I agree that using `off` and `on` everywhere would be messy. However, I'd say we're dealing with a somewhat special line of code, and using the `off` and `on` here means the file will be easier to keep well-formatted in the future.

Anyway, I am okay with either outcome.